src/ocean/util/log/Config.d

```diff
@@ -457,6 +466,18 @@ public void configureOldLoggers (
     Appender delegate ( istring file, Layout layout ) file_appender,
     bool use_insert_appender = false)
 {
+    // Using a dg literal as default argument in parameter list causes
+    // link failures in D1
+    configureOldLoggers(config, m_config, file_appender,
+        (cstring v) { return newLayout(v); }, use_insert_appender);
+}
+
+/// Ditto
+public void configureOldLoggers (
+    ClassIterator!(Config, ConfigParser) config, MetaConfig m_config,
+    Appender delegate ( istring file, Layout layout ) file_appender,
+    Layout delegate (cstring) makeLayout, bool use_insert_appender = false)
```

Comment:
The order of the last two parameters does not seem to correct for the overloading function. I think it should be the other way around (`use_insert_appender` should go first)